### ISSUE TITLE
Duress passcode?

### ISSUE THREAD

**Issue by Thunder33345** [keepassium/KeePassium#6]
A self destruct passcode that deletes all app db, app data, app settings TLDR
since clearing masterkey can already be done via protect database
but it would still be nice to have a separate self-destruct password with options of what to clear/keep

Not sure if possible due to secure enclave limitations, but also duress fingerprint to look even more legit?(from what i see, apple only tells your app if the authentication is successful or not)

**Comment by keepassium** [comment]
I agree, this would be nice to have. Thanks!

As for fingerprints, it would be impossible to distinguish different fingers: "Afterward, you receive only a Boolean result indicating authentication success or failure." [[docs](https://developer.apple.com/documentation/localauthentication)]

**Comment by Thunder33345** [comment]
Yeah i guessed, apple seems tight on touch ID, not even some kind of UUID is obtainable
I guess going to scrap the idea of "disabling fingerprint and force passcode if any fingerprints were changed" since it probably wont be possible to know if any was changed(i think appstore could tho(prompting password on first) having system privileges)

Also thank you very much for the good work,
An actually updated keepass compliant password manager that supports integrated passwords is let's just say not easy to come across...

**Comment by S2Ler** [comment]
Must have feature
